docs/user/instrument-android-crashes-example.md

```diff
@@ -0,0 +1,255 @@
+# Instrumenting Android Crashes with Glean
+
+One of the things that we might want to collect some data on in the applications we create is crashes.  Every application will
+have them from time to time.  Sometimes they are the application’s fault, sometimes it’s the fault of the OS, but they happen
+and we usually want to know more about them so we can fix them (if possible).  So let’s take a look at how we can use Glean to
+instrument an Android application and record some data when it crashes.
+
+## Before We Start
+
+There are a few things that we need to have installed in order to get ready.  The biggest one of those is [Android Studio](https://developer.android.com/studio/),
+which (if you include the Android SDK) can take a little while to download and get installed.  Second, I’m going to recommend
+having the [Book of Glean](https://mozilla.github.io/glean/book/) open and available as we will be referring to it throughout
+this walk-through as it contains all the Glean documentation we need to instrument our app.  This also assumes that you have
+some knowledge of Android application development.  We won’t be doing anything too involved, but knowing where to go to
+create a new project, and how to add dependencies to a Gradle file will be helpful.
+
+## Create A Project
+
+With our tools and documentation installed and open, let’s start by creating a new Android Studio project.  It could be any
+type of project to start with, but I’m choosing the “Basic Activity” template simply because it comes with a button already on
+the screen that can use to crash our app in order to collect our metrics.  Give the project a name of "GleanCrashExample",
+choose Kotlin for the language, and choose API level 21 for the minimum SDK level.
+
+## Setup Build Configuration
+
+### Add Glean As A Dependency
+
+Now that we have a project started, we can add Glean to it as a dependency in the Gradle configuration.  Find the
+`build.gradle` file for your _app module_ and add the following line to the dependencies section:
+
+```Groovy
+implementation 'org.mozilla.components:service-glean:16.0.0'
+```
+
+As of this writing, version `16.0.0` is the latest version of [Android Components](https://github.com/mozilla-mobile/android-components/).  
```

Comment:
Maybe change this to state where the current version can be looked up.
This part will _always_ be out of date.